### ISSUE TITLE
Transmitter does not copy the SF list into the Simulator 1.5.11 

### ISSUE THREAD

**Issue by Richrfl** [FrSkyRC/ETHOS-Feedback-Community#4318]
When copying  a model from the X18 TX (V 1.5.11 FCC) to the sim (users>name>.ethos18S>models the SF list is not copied (I guess all SFs are stored in the model.bin file). The SF list I can see in the sim is either SF that I created in the sim, or from a previous version of the TX FW.
Any comment, please.

**Comment by Nicholas-Luoyi** [comment]
It should load. Have you switched to the copied model?

**Comment by Richrfl** [comment]
Hi, Nick. I don't know if I did it right, but this is how I did it: I selected the model in the sim, cloned, selected the cloned model and the SFs were the same that I had in the original. The strange thing is that all LSs and telemetry sensors seem to be correct, but not the SFs. 
I remember I read something about copying the model in a thread related to the sim, but I think that was related to a different problem.
